### PR TITLE
Update software_list_s.md

### PR DESCRIPTION
**Pull request by anssi-cvp**
Updated Stormshield with specific advisory for StormShield Visibility Center (only product announced vulnerable to CVE-2021-44228)
Updated SonicWall product list based on v2.3 of their advisory

Thank you for your contribution! Some pointers to get it merged quickly:

For contributions in `software/`:

  - [x] PR Title: Please use "Add <vendor/product name>" (instead of "Update README.md")
  - [x] Status: please select a value from the status table at the top
  - [x] Version: Status Vulnerable / Workaround? -> List vulnerable versions.
             Status Fix?                     -> List fixed versions.
  - [x] Links: make sure you link to a public statement by the developer.
    Alternatively, include and link a file in the `software/vendor-statements/`
    directory
  - [x] Please mind the sorting

### CODE REVIEW

**Comment by tschmidtb51** [comment]
Just closing and reopening to trigger Github Actions.

File: software/software_list_s.md [hunk 1]
```diff
@@ -276,18 +277,21 @@
...
-|SonicWall|Analytics| | |Investigation| | | |[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
+|SonicWall|Analytics| |Not vuln|Not vuln|Not vuln|Not vuln|update based on v2.3 of advisory |[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
 |SonicWall|Analyzer| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
 |SonicWall|Capture Client & Capture Client Portal| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
 |SonicWall|Capture Security Appliance| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
 |SonicWall|CAS| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
-|SonicWall|Email Security|10.0.12|Not vuln|Fix| | | |[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
+|SonicWall|CSCMA| |Not vuln|Not vuln|Not vuln|Not vuln|based on v2.3 of advisory|[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
```
Comment (govgitty) [inline software/software_list_s.md:285]: CVE-2021-4104 is not mentioned in the advisory.
```suggestion
|SonicWall|CSCMA| | |Not vuln|Not vuln|Not vuln|based on v2.3 of advisory|[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|

File: software/software_list_s.md [hunk 2]
```diff
@@ -276,18 +277,21 @@
...
+|SonicWall|Cloud Edge| |Not vuln|Not vuln|Not vuln|Not vuln|based on v2.3 of advisory|[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
```
Comment (govgitty) [inline software/software_list_s.md:286]: CVE-2021-4104 is not mentioned in the advisory.
```suggestion
|SonicWall|Cloud Edge| | |Not vuln|Not vuln|Not vuln|based on v2.3 of advisory|[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|

File: software/software_list_s.md [hunk 3]
```diff
@@ -276,18 +277,21 @@
...
+|SonicWall|Email Security|10.0.13|Not vuln|Fix|Fix|Fix|based on version 2.3 of advisory|[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
```
Comment (govgitty) [inline software/software_list_s.md:287]: CVE-2021-4104 is not mentioned in the advisory.
```suggestion
|SonicWall|Email Security|10.0.13| |Fix|Fix|Fix|based on version 2.3 of advisory|[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|

File: software/software_list_s.md [hunk 4]
```diff
@@ -276,18 +277,21 @@
...
+|SonicWall|EPRS| |Not vuln|Not vuln|Not vuln|Not vuln|based on version 2.3 of advisory|[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
```
Comment (govgitty) [inline software/software_list_s.md:288]: CVE-2021-4104 is not mentioned in the advisory.
```suggestion
|SonicWall|EPRS| | |Not vuln|Not vuln|Not vuln|based on version 2.3 of advisory|[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|

File: software/software_list_s.md [hunk 5]
```diff
@@ -276,18 +277,21 @@
...
 |SonicWall|Gen5 Firewalls (EOS)| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
 |SonicWall|Gen6 Firewalls| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
 |SonicWall|Gen7 Firewalls| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
 |SonicWall|GMS| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
 |SonicWall|MSW| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
-|SonicWall|NSM| |Not vuln|Fix| | |SonicWall automatically pushed a new NSM firmware|[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032#:~:text=NSM%202.x%20(OnPrem%2C%20SaaS))|
+|SonicWall|NSM On-Premise|2.3.2-R12-H2|Not vuln|Fix|Fix|Fix|based on version 2.3 of advisory|[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|
```
Comment (govgitty) [inline software/software_list_s.md:294]: CVE-2021-4104 is not mentioned in the advisory.
```suggestion
|SonicWall|NSM On-Premise|2.3.2-R12-H2| |Fix|Fix|Fix|based on version 2.3 of advisory|[source](https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032)|

**Review by govgitty — CHANGES_REQUESTED** [review]
hi @anssi-cvp , thanks for contributing to the software list. Can you have a look at my comments? Some changes are necessary before we can merge.

**Comment by anssi-cvp** [comment]
Hello, the original line seems to have one separator more than expected. What do you prefer ? Should I cancel my PR and send it again after correcting this error or do you do it on your side ?

**Comment by govgitty** [comment]
@anssi-cvp, thank you for your contribution.

**Comment by tschmidtb51** [comment]
> Hello, the original line seems to have one separator more than expected. What do you prefer ? Should I cancel my PR and send it again after correcting this error or do you do it on your side ?

@anssi-cvp You can always add commits to a Pull Request (to be precise: to the branch the PR is based on) to fix such issues by yourself. There is no need to cancel a PR if a check fails.

**Pull request merged**